ci-operator/templates/openshift/installer/cluster-disaster-control-plane-ipi.yaml

```diff
@@ -0,0 +1,501 @@
+kind: Template
```

Comment:
nit with the filename, I think we can assume IPI and only call out UPI in UPI templates (which is our current pattern), so I'd prefer `cluster-disaster-control-plane.yaml`.